modules/ossm-load-test-results.adoc

```diff
@@ -52,7 +52,7 @@ The latency, throughput, and the proxies' CPU and memory consumption are measure
 
 === CPU and memory consumption
 
-Since the sidecar proxy performs additional work on the data path, it consumes CPU and memory. As of Istio 1.1, a proxy consumes about 0.6 vCPU per 1000 requests per second.
+Since the sidecar proxy performs additional work on the data path, it consumes CPU and memory. As of Istio 1.12.3, a proxy consumes about 0.5 vCPU per 1000 requests per second.
```

Comment:
https://istio.io/latest/docs/ops/deployment/performance-and-scalability/#cpu-and-memory

Here, the Istio version is 1.14.1

Reply:
@pbajjuri20 Make sure to look at the upstream 1.12 doc page: https://istio.io/v1.12/docs/ops/deployment/performance-and-scalability/